tests/render/test_inline_renderer.py

```diff
@@ -9,11 +9,29 @@
     ExpectationValidationResult,
 )
 from great_expectations.data_context import DataContext
+from great_expectations.render.exceptions import InvalidRenderedContentError
 from great_expectations.render.renderer.inline_renderer import InlineRenderer
 from great_expectations.render.types import RenderedAtomicContent
 from great_expectations.validator.validator import Validator
 
 
+@pytest.mark.unit
+def test_inline_renderer_error_message(
+    alice_columnar_table_single_batch_context: DataContext,
```

Comment:
This fixture takes quite some time to put together due to some I/O - could we use a simpler mock context to keep this a unit test?

Reply:
I haven't been able to locate a mock context or expectation suite. I think we should table this until we can mock more of our core objects. I have labeled this as integration.